.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: build
+
+on:
+  release:
+    types:
+      - published
```

Comment:
this script builds the wheel, packages the source, and uploads to PyPI upon publish of a new release, using the release tag